content/docs/user-guide/assets/asset-processor/interface.md

```diff
@@ -165,37 +165,41 @@ You can browse source assets in the scan directories to view IDs and dependency
 
 With an asset selected, you can **right-click** and select from several actions such as view the source asset, view the job, open the asset location in a file manager, copy the asset path, and reprocess the asset.
 
-![Asset Processor UI assets tab](/images/user-guide/assets/asset-processor/interface-assets.png)
-
 ### Source Assets
 
-On the left, the Source Assets list displays all the source assets that match the current search filter. When the search is empty, all source assets are displayed.
+On the left, the Source Assets tree displays all the source assets that match the current search filter. When the search is empty, all source assets are displayed. The "Last Analysis Job Duration" column shows the accumulation of duration values of all builders' CreateJobs on this source asset.
+
+When an asset is selected in the Source Assets list, information about the asset is displayed in the header and tabs on the right.
 
-When an asset is selected in the Source Assets list, information about the asset is displayed in the panes on the right.
+![Asset Processor UI assets tab - source assets](/images/user-guide/assets/asset-processor/interface-assets-source.png)
 
 | Pane | Description |
 | - | - |
 | **Asset Information** | Detailed information about the selected asset including the name of the asset, the scan directory path, and the Universally Unique Identifier (UUID) associated with the asset. |
 | **Products** | The product assets that are produced from jobs that processed this source asset. |
-| **Outgoing Source Dependencies** | Any files that have been registered as a source dependency for any jobs that run on this source asset. |
-| **Incoming Source Dependencies** | The list of source assets that have one or more jobs that have marked this source asset as a source dependency. Any modifications to this source asset will cause these jobs to run. |
+| **Dependencies - Out** | Any files that have been registered as a source dependency for any jobs that run on this source asset. |
+| **Dependencies - In** | The list of source assets that have one or more jobs that have marked this source asset as a source dependency. Any modifications to this source asset will cause these jobs to run. |
 
 {{< note >}}
 Choose the {{< icon "open-in-internal-app.svg" >}} **Go to** icon next to a product asset name to go to that asset in the Product Assets tab, or this icon next to a source asset name to go to that asset in the Source Assets tab.
 {{< /note >}}
 
+### Intermediate Assets

```

Comment:
was there supposed to be some text for this heading?

Reply:
There are Intermediate Assets and Shared Cached in the screenshot, so I think the document has to have these sections. However, since I am not the best person to describe the changes in Intermediate Assets and Shared Cache, I decided to leave them as not-in-scope future work. 

Or do you mean placing some placeholders in these sections?

Reply:
added short paragraph to intermediate asset and shared cache sections